CHANGELOG.rst

```diff
@@ -27,6 +27,7 @@ Fixed
 ~~~~~
 
 - (`#36 <https://github.com/openscm/openscm-runner/pull/36>`_) Hotfix CI after pandas 1.1.5 broke pylint
+- (`#37 <https://github.com/openscm/openscm-runner/pull/33>`_) Stop FaIR trying to run aggregated CO2 emissions
```

Comment:
```suggestion
- (`#37 <https://github.com/openscm/openscm-runner/pull/33>`_) Stop FaIR trying to run with total CO2 emissions
```

Trying to think of a clearer way to explain the change

Reply:
Changed it to 

- (`#37 <https://github.com/openscm/openscm-runner/pull/33>`_) Ensure FaIR ignores emissions input in scenarios not handled by FaIR, e.g. total CO2

Hope that's better!